docs-web/configuration/virtualserver-and-virtualserverroute-resources.md

```diff
@@ -507,6 +507,10 @@ tls:
      - Sets the size of the buffer used for reading the first part of a response received from the upstream server. See the `proxy_buffer_size <https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_buffer_size>`_ directive. The default is set in the ``proxy-buffer-size`` ConfigMap key.
      - ``string``
      - No
+   * - ``use-cluster-ip``
```

Comment:
would it make sense to place it under service or subselector? Cause those options are related. this way for a user it would be easier to locate the `use-cluster-ip` field.